clcache.py

```diff
@@ -448,15 +451,18 @@ def save(self):
 class AnalysisResult:
     Ok, NoSourceFile, MultipleSourceFilesSimple, \
         MultipleSourceFilesComplex, CalledForLink, \
-        CalledWithPch, ExternalDebugInfo = range(7)
+        CalledWithPch, ExternalDebugInfo = list(range(7))
 
 
-def getFileHash(filePath, additionalData=None):
+def getFileHash(filePath, additionalDataString=None):
     hasher = HASH_ALGORITHM()
     with open(filePath, 'rb') as inFile:
         hasher.update(inFile.read())
-    if additionalData is not None:
-        hasher.update(additionalData)
+    if additionalDataString is not None:
```

Comment:
I'm not really fan of hungarian notation, I'd rather not merge this renaming work (which doesn't seem to be needed to fulfill the purpose of this commit, implementing support for Python 2.7 and Python 3.3).